taxonomies/labels.txt

```diff
@@ -573,207 +579,6 @@ de:FSC Recycling
 fi:FSC-kierrätys
 fr:FSC Recyclé, FSC Recyclage
 
-<en:FSC
```

Comment:
Why do you want to remove the FSC labels and how does that change relate to clustering?

Reply:
unrelated. They were moved to packager codes, and this was a really tiny subset of all the FSC codes.

Reply:
@hangy should I remove them in a separate PR or are you ok with getting rid of them in this one ?